tests/unit/components/__snapshots__/HdTileSelect.spec.js.snap

```diff
@@ -0,0 +1,43 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`HdTileSelect allows custom new value 1`] = `
+<section class="field tile-select">
+  <section class="tile-select__items"><button type="button" class="tile-select-item tile-select__item">
+      1
+    </button><button type="button" class="tile-select-item tile-select__item">
+      2
+    </button><button type="button" class="tile-select-item tile-select__item">
+      3
+    </button> <button type="button" class="tile-select-item tile-select-editable-item tile-select__item">
```

Comment:
Seeing the snapshot build now feels like sth. goes wrong, as we have three classes on the tag. I cannot see
any longer how the parentage is defined with 2 blocks and one element. Same for the other lines and HdTileSelectEditableItem.spec.js.snap

Reply:
We have this issue as we're scoping each component independently so it will have such conflicts. Doing something like I mentioned here (https://github.com/homeday-de/homeday-blocks/pull/465#discussion_r444054555) might help as we will have a wrapper to deal with base styles.